docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Health.adoc

```diff
@@ -155,6 +169,12 @@ The MicroProfile Config property `mp.health.disable-default-procedures` is read
 2. __When an application is deployed__, to determine if WildFly should add a readiness check if the deployment does not provide any. At that time, setting this property in a `microprofile-config.properties` file in the deployment would be taken into account. (with the usual priority rules for MicroProfile Config properties).
 ====
 
+When the `mp.health.disable-default-procedures` is set to `true` the server will not include any of its health checks in the responses which included also the default empty configurable checks indluded before the server deploys the deployments, namely `empty-readiness-checks`, `empty-startup-checks`, and `empty-liveness-checks`. This means that the server might prematurely responds with invalid `UP` response particularly to `startup` and `readiness` invocations before the user deployment is processed. For this reason, MicroProfile Health specifications defines two MicroProfile Config properties that specify the response returned while the server is still processing deployments which means it returns and empty health response:
```

Comment:
Can we reword the initial sentence to avoid repeating `include`? Something like:
```
When the `mp.health.disable-default-procedures` is set to `true`,  the server will not return any of the health checks which contain the default empty configurable checks included before the deployments are processed, namely `empty-readiness-checks`, `empty-startup-checks`, and `empty-liveness-checks`.
```
WDYT?

BTW - 
* s/prematurely responds/prematurely respond/ 
* s/MicroProfile Health specifications defines/MicroProfile Health specifications define/
* s/processing deployments which means it returns and/processing deployments, i.e. it returns an/
?

Reply:
I see you addressed my comment here. Thanks, the thread can be resolved.